packages/health-and-info/src/index.ts

```diff
@@ -0,0 +1,195 @@
+import AgentOptions, { HttpOptions, HttpsAgent, HttpsOptions } from 'agentkeepalive'
+import superagent from 'superagent'
+import { Request, Response, NextFunction } from 'express'
+import {
+  ComponentCheckResult,
+  ExternalComponent,
+  CustomComponent,
+  ApplicationInfo,
+  HealthCheckOptions,
+  HealthCheckResult,
+} from './types/HealthCheckOptions'
+
+export default class HealthCheck {
+  private defaultOptions = {
+    components: {
+      retries: 2,
+      timeout: {
+        response: 1500,
+        deadline: 2000,
+      },
+      enabled: true,
+    },
+  }
+
+  constructor(
+    private readonly options: HealthCheckOptions,
+    private readonly logger: Console = console,
```

Comment:
**suggestion:** would it be better to not to provide a default?
To encourage people to hook up app insights?

Reply:
Hmm... I'm a little torn, on the one hand, I like module's not needing unnecessary config and parameters, but on the other hand, we could be leading to bad practice - and as you say, letting teams skip connecting up app-insights/a proper logger.

Let's strip it out. I need to add a comment on the constructor anyway that makes it more clear how it's meant to be called, here I can mention to pass a more succinct logger, and if not, just suggest they use Console as their last option.